gulpfile.js

```diff
@@ -97,7 +97,9 @@ function dist_css_build () {
     const paths = [
         'node_modules/bootstrap/dist/css/bootstrap.min.css',
         'node_modules/ui-select/dist/select.min.css',
-        'node_modules/angular-bootstrap-datetimepicker/src/css/datetimepicker.css',
+        'node_modules/bootstrap-datepicker/dist/css/bootstrap-datepicker.min.css',
+        'node_modules/angularjs-bootstrap-datetimepicker/src/css/datetimepicker.css',
+        'node_modules/angularjs-bootstrap-datetimepicker/src/js/datetimepicker.templates.js',
```

Comment:
This JS file should be with the other JS files.

Reply:
When you make changes to gulpfile.js, make sure to run `gulp` and commit the changes.